apps/site/src/content/blog/state-of-the-budgie-2023.mdx

```diff
@@ -0,0 +1,203 @@
+
+#### Magpie v0.x
+
+During our announcement post related to our Wayland plans, we further announced our Budgie-specific fork of Mutter as it was in GNOME 43. The goal of this was to address regressions that were, understandably, incrementally introduced into Mutter related to X11 support. These regressions created a wide-range of issues for users and it was determined that our engineering time would be best spent focusing on Wayland rather than fixing up X11-specific bugs and updating Budgie to support new Mutter ABI changes. Magpie v0.x provides us a stable ABI that can be leveraged across multiple GNOME Stack releases until Magpie 1.0 (built on wlroots) is released.
+
+Budgie 10.8 was the first release to start requiring Magpie v0.x and the result has been a more consistent and reliable experience for our users.
+
+## Wayland efforts
+
+While our goal in 2023 was only to start experimenting with Wayland compositors and to take the learnings from those experiments to apply to a "future Budgie WM compositor" built for Budgie 11, we made a significant shift in our development focus starting middle of last year to accelerate our Wayland plans and work towards Budgie 10 being Wayland-only (and continuing to be Wayland-only going into Budgie 11). This was in recognition of several major developments in the broader ecosystem that we covered in our [Wayland](/wayland) post.
+
+In that lengthy post, we covered our experiences at that point with wlroots as well as XFCE's libxfce4windowing library, which provides us a library which supports both X11 (via libwnck) and Wayland (various Wayland protocols) at the same time. This is especially useful as we iterate on making Budgie Wayland-compatible while using it at the same time.
+
+I'm happy to report that progress on Wayland support has been steady on all fronts since our shift in development focus to it.
+
+In Budgie Desktop, several applets and UX components of Budgie Desktop have been ported over or are in the process of being ported:
+
+- Show Desktop Applet has been ported
+- Tab Switcher has been ported
+- Workspaces Applet has been ported
+- IconTasklist Applet is currently being ported.
+
+Keep in mind that not every part of the codebase needs to be touched, but some key areas in the codebase will need to be updated once some newer window tracking logic has been merged in from the IconTasklist conversion: MenuManager (right-click menus), Panel Manager (manages panels), the legacy Tasklist needs to be rewritten, and various uses of Gdk.X11 APIs.
+
+On the Magpie v1.0 front, the progress by Campbell Jones (aka serebit) has been fantastic as well:
+
+- @serebit write things here please :)
```

Comment:
@serebit if you could provide a suggestion as part of a review to replace this with your content, or just commit a write-up of your Magpie work that you would like to discuss, that would be greatly appreciated!

Reply:
We should include the effort here by Neal to actually allow wayland budgie to be launched via a login manager such as lightdm.  This will open up opportunities such as creating a budgie specific portal thus providing better support for flatpaks and snaps (for example)

Reply:
This is for the 2023 section. There hasn't been anything by Neal in 2023 on that front. 2024 has information about getting Budgie running in a Wayland section has a target.